src/main/java/org/spongepowered/common/mixin/core/item/MixinItemArmorMaterial.java

```diff
@@ -58,4 +67,23 @@ public String getId() {
     public String getName() {
         return this.name;
     }
+
+    @Override
+    public Optional<ItemType> getRepairItemType() {
```

Comment:
I cannot find the relevant code in forge can you link it here please?

IMO we should first fix it here and we can also backport this as is and then decide how to implement it in forge differently.